modules/convert/issue.go

```diff
@@ -5,10 +5,96 @@
 package convert
 
 import (
+	"strings"
+
 	"code.gitea.io/gitea/models"
 	api "code.gitea.io/gitea/modules/structs"
 )
 
+// ToAPIIssue converts an Issue to API format
+// it assumes some fields assigned with values:
+// Required - Poster, Labels,
+// Optional - Milestone, Assignee, PullRequest
+func ToAPIIssue(issue *models.Issue) *api.Issue {
+	if err := issue.LoadLabels(); err != nil {
```

Comment:
It seems that you could just call `issue.LoadAttributes()`. It also loads the assignees, but I don't think that's a problem.

Reply:
I just moved this function, didn't want to change any behaviour